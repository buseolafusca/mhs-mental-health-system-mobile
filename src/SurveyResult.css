.SurveyResult {
  text-align: center;
}

.SurveyResult-logo {
  animation: SurveyResult-logo-spin infinite 20s linear;
  height: 40vmin;
  pointer-events: none;
}

.SurveyResult-header {
  background-color: #282c34;
  min-height: 100vh;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  font-size: calc(10px + 2vmin);
  color: white;
}

.SurveyResult-link {
  color: #61dafb;
}

@keyframes SurveyResult-logo-spin {
  from {
    transform: rotate(0deg);
  }
  to {
    transform: rotate(360deg);
  }
}
