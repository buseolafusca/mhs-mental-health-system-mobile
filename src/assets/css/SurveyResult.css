.SurveyResult {
  background-color: #F1F1F1;
}

.sv_row{
  text-align: left;
}

.SurveyResult-logo {
  animation: SurveyResult-logo-spin infinite 20s linear;
  height: 40vmin;
  pointer-events: none;
}

.SurveyResult-header {
  background-color: #282c34;
  min-height: 100vh;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  font-size: calc(10px + 2vmin);
  color: white;
}

.SurveyResult-link {
  color: #61dafb;
}

.box1 {
  background-color: #005eb8;
  color: #fff;
  border-radius: 5px;
  text-align: center;
  padding: 5px 0px 5px 0px;
  font-size: 100%;
}

@keyframes SurveyResult-logo-spin {
  from {
    transform: rotate(0deg);
  }
  to {
    transform: rotate(360deg);
  }
}
