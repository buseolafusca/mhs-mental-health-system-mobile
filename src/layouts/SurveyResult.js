import React from 'react';
import logo from './logo.svg';
import './SurveyResult.css';

function SurveyResult() {
  return (
    <div className="SurveyResult">
      <header className="SurveyResult-header">
        <img src={logo} className="SurveyResult-logo" alt="logo" />
        <p>
          Edit <code>src/SurveyResult.js</code> and save to reload.
        </p>
        <a
          className="SurveyResult-link"
          href="https://reactjs.org"
          target="_blank"
          rel="noopener noreferrer"
        >
          Learn React
        </a>
      </header>
    </div>
  );
}

export default SurveyResult;
