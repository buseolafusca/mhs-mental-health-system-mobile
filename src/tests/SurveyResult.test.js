import React from 'react'
import ReactDOM from 'react-dom'
import SurveyResult from '../layouts/SurveyResult'
import { shallow } from 'enzyme'

it('renders without crashing', () => {

  const match = { params: { id: '5d4abc0139a0a411900f7b42' } }
  const component = shallow(<SurveyResult match={match} />)
  expect(component).toMatchSnapshot()

})
