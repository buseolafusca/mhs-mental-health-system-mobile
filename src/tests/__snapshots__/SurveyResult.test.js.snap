// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  id="page-container"
>
  <NHSHeader />
  <div
    className="SurveyResult"
  >
    <div
      className="surveyjs"
    >
      <Survey
        model={
          Object {
            "mode": "display",
            "showProgressBar": "",
          }
        }
      />
      <div>
         
        <Square
          title="Back"
          url="/landingpage"
        />
         
      </div>
    </div>
  </div>
  <NHSFooter />
</div>
`;
